README.md

```diff
@@ -11,14 +11,33 @@ If you work a with a git rebase flow, git machete will (vastly!) help you manage
 
 ## Install
 
-Run the following commands to install git machete:
+Installation could be done by following commands:
 
 ```bash
 $ git clone https://github.com/VirtusLab/git-machete.git
 $ cd git-machete
-$ sudo make install
+$ sudo python setup.py install
 ```
 
+To install you will need python with installed `pip` and `setuptools`.
+
+python could be installed with system packages or using [pyenv](https://github.com/pyenv/pyenv).
+
+Regular python env should have `pip` and `setuptools`.
+
+If python env is missing any of those tools, please execute following commands:
+
+```bash
+# missing pip? check with command: pip
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
```

Comment:
`Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/usr/local/lib/python2.7/dist-packages/pip'
Consider using the `--user` option or check the permissions.`

So maybe... change to `python get-pip.py  --user` instead?

Reply:
True. That depends on how python env is installed.